airflow/metrics/otel_logger.py

```diff
@@ -394,8 +395,11 @@ def get_otel_logger(cls) -> SafeOtelLogger:
     allow_list_validator = AllowListValidator(allow_list)
 
     resource = Resource(attributes={SERVICE_NAME: "Airflow"})
-    # TODO:  figure out https instead of http ??
-    endpoint = f"http://{host}:{port}/v1/metrics"
+
+    if ssl_active:
+        endpoint = f"https://{host}:{port}/v1/metrics"
+    else:
+        endpoint = f"http://{host}:{port}/v1/metrics"
```

Comment:
I'd either use 

```suggestion
    endpoint = f"{'https' if ssl_active else 'http'}://{host}:{port}/v1/metrics"
```


or

```suggestion
    protocol = "https" if ssl_active else "http"
    endpoint = f"{protocol}://{host}:{port}/v1/metrics"
```

Reply:
thanks @ferruzzi, I have updated the code